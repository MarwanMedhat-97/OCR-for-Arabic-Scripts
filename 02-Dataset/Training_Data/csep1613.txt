اختتم مس منتخبنا الوطني الول لكر القدم معسكره الاعدادي الول استعدادا لمباراته المرتقبه مع نظيره الياباني يوم الثالث عشر من اكتوبر المقبل في طار منافسات تصفيات المجموع السيوي الثالث المهل ال مونديال وقد اقتصر هذا المعسكر عل التركيز عل رفع معدل اللياق البدني لد اللاعبين وتطبيق بعض التمارين التكتيكي في الهجوم والدفاع بالاضاف ال تجرب العناصر الجديد التي انضمت لهذا المعسكر لول مر بهدف التعرف عل مستوياتهم الفني والبدني قبل الانضمام رسميا ال التشكيل الساسي للمنتخب وكان ماتشالا قد بد انزعاجه من مستو اللياق البدني لد اللاعبين وعدم انتظامهم في تدريبات نديتهم ولذلك تم التركيز في هذا المعسكر عل هذا الجانب حيث لم يخض المنتخب خلاله ي مباريات ودي بل اكتف بالتمارين والتي قيمت عل فترتين صباحي ومساي عل الملعب الفرعي بمجمع السلطان قابوس الرياضي ببوشروسوف يعود المنتخب يوم السبت القادم للدخول في المعسكر الاعدادي الثاني الذي سيقام خلال الفتر من ال اكتوبر المقبل سيخوض خلاله مبارا ودي واحد مع منتخب العراقي الشقيق يوم اكتوبر المقبل الذي يخوض هو الخر مبارا حاسم مام اوزبكستان في الثالث عشر من اكتوبر المقبل ضمن تصفيات المجموع السيوي الثاني المهل ال مونديال وقد شارك في تدريبات هذا المعسكر لاعبا هم حسن ربيع وعلي بن طالب العجمي وبدر المحروقي ومحمد بن مبارك الهناي وسلطان الطوقي ومحمد بن حمد واحمد بن مبارك المحيجري وحسين بن مستهيل واحمد بن حديد وفوزي بن بشير دوربين وهاشم بن صالح ويوسف شعبان واحمد سالم وحسن بن يوسف المظفر ويعقوب بن ربيع وحسن العجمي وعبدالله سعيد وجاسم يعقوب وهاني الضابط الذي خاض مع المنتخب تدريبات تهيلي وتقوي وذلك للاطمنان عل شفاه من الاصاب التي يعاني منها حيث سيقوم يوم السبت القادم بالسفر ال بوظبي لمقابل البرفيسور الذي اشرف عل العملي الجراحي التي جراها قبل ست شهور وذلك لخوض الكشف النهاي عل الاصاب وصول المحترفين السبت المقبليصل يوم السبت القادم لاعبو منتخبنا المحترفون في الخارج للانضمام ال صفوف المنتخب يوم الاثنين المقبل وذلك للمشارك في المعسكر الثاني الذي سيقام خلال الفتر من وحت اكتوبر المقبل وسيخوض خلاله مبارا ودي واحد مع منتخب العراق يوم من الشهر القادم واللاعبون هم بدر الميمني وحمدي هوبيس وسعيد الشون وعماد بن علي الحوسني ومحمد بن ربيع وخليف عايل بينما سيصل علي الحبسي يوم اكتوبر المقبل