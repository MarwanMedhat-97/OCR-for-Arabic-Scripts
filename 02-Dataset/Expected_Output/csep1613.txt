 اختتم أمس منتخبنا الوطني الأول لكرة القدم معسكره الاعدادي الأول استعدادا لمباراته المرتقبه مع نظيره الياباني يوم الثالث عشر من اكتوبر المقبل في إطار منافسات تصفيات المجموعة الآسيوية الثالثة المؤهلة الى مونديال 2006 .وقد اقتصر هذا المعسكر على التركيز على رفع معدل اللياقة البدنية لدى اللاعبين وتطبيق بعض التمارين التكتيكية في الهجوم والدفاع بالاضافة الى تجربة العناصر الجديدة التي انضمت لهذا المعسكر لأول مرة بهدف التعرف على مستوياتهم الفنية والبدنية قبل الانضمام رسميا الى التشكيلة الأساسية للمنتخب وكان ماتشالا قد أبدى انزعاجه من مستوى اللياقة البدنية لدى اللاعبين وعدم انتظامهم في تدريبات أنديتهم ولذلك تم التركيز في هذا المعسكر على هذا الجانب حيث لم يخض المنتخب خلاله أي مباريات ودية بل اكتفى بالتمارين والتي أقيمت على فترتين صباحية ومسائية على الملعب الفرعي بمجمع السلطان قابوس الرياضي ببوشر.وسوف يعود المنتخب يوم السبت القادم للدخول في المعسكر الاعدادي الثاني الذي سيقام خلال الفترة من 2 الى 13 اكتوبر المقبل سيخوض خلاله مباراة ودية واحدة مع منتخب العراقي الشقيق يوم 7 اكتوبر المقبل الذي يخوض هو الآخر مباراة حاسمة أمام اوزبكستان في الثالث عشر من اكتوبر المقبل ضمن تصفيات المجموعة الآسيوية الثانية المؤهلة الى مونديال 2006.وقد شارك في تدريبات هذا المعسكر 19 لاعبا هم : حسن ربيع وعلي بن طالب العجمي وبدر المحروقي ومحمد بن مبارك الهنائي وسلطان الطوقي ومحمد بن حمد واحمد بن مبارك المحيجري وحسين بن مستهيل واحمد بن حديد وفوزي بن بشير دوربين وهاشم بن صالح ويوسف شعبان واحمد سالم وحسن بن يوسف المظفر ويعقوب بن ربيع وحسن العجمي وعبدالله سعيد وجاسم يعقوب وهاني الضابط الذي خاض مع المنتخب تدريبات تأهيلية وتقوية وذلك للاطمئنان على شفائه من الاصابة التي يعاني منها حيث سيقوم يوم السبت القادم بالسفر الى أبوظبي لمقابلة البرفيسور الذي اشرف على العملية الجراحية التي أجراها قبل ستة شهور وذلك لخوض الكشف النهائي على الاصابة .وصول المحترفين السبت المقبليصل يوم السبت القادم لاعبو منتخبنا المحترفون في الخارج للانضمام الى صفوف المنتخب يوم الاثنين المقبل وذلك للمشاركة في المعسكر الثاني الذي سيقام خلال الفترة من 2 وحتى 13 اكتوبر المقبل وسيخوض خلاله مباراة ودية واحدة مع منتخب العراق يوم 7 من الشهر القادم .واللاعبون هم: بدر الميمني وحمدي هوبيس وسعيد الشون وعماد بن علي الحوسني ومحمد بن ربيع وخليفة عايل بينما سيصل علي الحبسي يوم 5 اكتوبر المقبل .
